kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/thirdChallenge/FlightFinder.java

```diff
@@ -51,26 +32,16 @@ private void scanFlightsFromAirport() {
 
             flightsFormChosenAirport
                     .stream()
-                    .map(result -> result.toString())
+                    .map(Flight::toString)
```

Comment:
czy to konieczne? klasa Flight ma metodę toString(), która jest automatycznie przez Javę wywoływana, gdy robimy print